recipes-test/tegra-tests/nvidia-docker-tests_1.0.bb

```diff
@@ -0,0 +1,44 @@
+DESCRIPTION = "Scripts for testing docker containers with PyTorch, TensorFlow, TensorRT and CUDA library"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE.md;md5=da33f1c12f6d31e71cd114d0c336d4be"
+
+SRC_REPO ?= "github.com/atharvanan1/jetson-containers"
```

Comment:
@madisongh should this live in an OE4T repo before https://github.com/dusty-nv/jetson-containers/pull/189 is approved?

Reply:
We can shift it there. If you can stage it there, I'll change the SRC_REPO accordingly.

Reply:
The changes look pretty small... any chance you could just generate a patch to apply to the upstream sources until the PR is resolved?

Reply:
Noted! Thanks for you input.

Reply:
Done.